go.mod

```diff
@@ -421,3 +421,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/rancher/shepherd => github.com/susesgartner/shepherd v0.0.0-20240307220711-9970e2c20892
```

Comment:
Since your testing is done with Jenkins, please update this to not point to your forked repo.

Reply:
Will do.